.devcontainer/vscode/tasks.json

```diff
@@ -37,100 +85,104 @@
-			"clear": true,
-			"revealProblems": "onProblem"
+			"options": {
+				"cwd": "${workspaceFolder}/src"
+			}
 		},
-		"options": {
-			"cwd": "${workspaceFolder}/src"
-		}
-	  },
-	  {
-		"label": "Maintenance: manage.py createsuperuser",
-		"type": "shell",
-		"command": "pipenv run python manage.py createsuperuser",
-		"group": "none",
-		"presentation": {
-			"echo": true,
-			"reveal": "always",
-			"focus": true,
-			"panel": "shared",
-			"showReuseMessage": false,
-			"clear": true,
-			"revealProblems": "onProblem"
+		{
+			"label": "Maintenance: manage.py createsuperuser",
+			"type": "shell",
+			"command": "pipenv run python manage.py createsuperuser",
+			"group": "none",
+			"presentation": {
+				"echo": true,
+				"reveal": "always",
+				"focus": true,
+				"panel": "shared",
+				"showReuseMessage": false,
+				"clear": true,
+				"revealProblems": "onProblem"
+			},
+			"options": {
+				"cwd": "${workspaceFolder}/src"
+			}
 		},
-		"options": {
-			"cwd": "${workspaceFolder}/src"
-		}
-	  },
-	  {
-		"label": "compile frontend",
-		"type": "shell",
-		"command": "npm ci && ./node_modules/.bin/ng build --configuration production",
-		"group": "none",
-		"presentation": {
-			"echo": true,
-			"reveal": "always",
-			"focus": true,
-			"panel": "shared",
-			"showReuseMessage": false,
-			"clear": true,
-			"revealProblems": "onProblem"
+		{
+			"label": "Maintenance: Compile frontend for production",
+			"type": "shell",
+			"command": "npm ci && ./node_modules/.bin/ng build --configuration production",
+			"group": "none",
+			"presentation": {
+				"echo": true,
+				"reveal": "always",
+				"focus": true,
+				"panel": "shared",
+				"showReuseMessage": false,
+				"clear": true,
+				"revealProblems": "onProblem"
+			},
+			"options": {
+				"cwd": "${workspaceFolder}/src-ui"
+			}
 		},
-		"options": {
-			"cwd": "${workspaceFolder}/src-ui"
-		}
-	  },
-	  {
-		"label": "Maintenance: recreate .venv",
-		"type": "shell",
-		"command": "rm -R -v .venv/* || pipenv install --dev",
-		"group": "none",
-		"presentation": {
-			"echo": true,
-			"reveal": "always",
-			"focus": true,
-			"panel": "shared",
-			"showReuseMessage": false,
-			"clear": true,
-			"revealProblems": "onProblem"
+		{
+			"label": "Maintenance: recreate .venv",
+			"type": "shell",
+			"command": "rm -R -v .venv/* || pipenv install --dev",
+			"group": "none",
+			"presentation": {
+				"echo": true,
+				"reveal": "always",
+				"focus": true,
+				"panel": "shared",
+				"showReuseMessage": false,
+				"clear": true,
+				"revealProblems": "onProblem"
+			},
+			"options": {
+				"cwd": "${workspaceFolder}"
+			}
 		},
-		"options": {
-			"cwd": "${workspaceFolder}"
-		}
-	  },
-	  {
-		"label": "Celery Worker",
-		"type": "shell",
-		"command": "pipenv run celery --app paperless worker -l DEBUG",
-		"group": {
-		  "kind": "build",
-		  "isDefault": true
+		{
+			"label": "Maintenance: Install Angular Frontend for Debug",
```

Comment:
```suggestion
			"label": "Maintenance: Install Frontend Dependencies",
```

Reply:
updated with commit "task for documentation build"